salt/tls/init.sls

```diff
@@ -25,3 +31,101 @@ ssl-cert:
     - require:
       - pkg: ssl-cert
 {% endfor %}
+
+# initial test
+{% if grains['id'] == 'salt.nyc1.psf.io' or grains['id'] == 'salt-master.vagrant.psf.io' %}
+pypa.io:
+  acme.cert:
+    - email: infrastructure-staff@python.org
+    - webroot: /etc/lego
+    - renew: 14
+    {% if pillar["dc"] == "vagrant" %}
+    - server: https://salt-master.vagrant.psf.io:14000/dir
```

Comment:
docs say there's built in support for using a test CA: with `test_cert`, which is probably the easiest way? https://docs.saltproject.io/en/3006/ref/states/all/salt.states.acme.html

Reply:
Ah, I see that just uses Lets Encrypt's test server, which still tries to validate.

Reply:
@JacobCoffee 0aefff26ea07e16abfa97cb90237eb862e6487cd should unblock you.